README.md

```diff
@@ -91,13 +91,13 @@ specific RuboCop release.**
 
 **RuboCop**'s installation is pretty standard:
 
-```
+```sh
```

Comment:
I think 'sh' and individual command line shell commands should be pretty much the same thing.

Somewhat tangentally, I'm not a fan of all the "$" prompt prefixes, as they hinder copy/paste :)